### PR TITLE
feature/Task-01-Frontend-Add-a-Nav-Bar

### PR DESCRIPTION
**Pull request by sayanta28**
## Why
feature/Task-01-Frontend-Add-a-Nav-Bar

## Insights
- Add ReadMe

## Proof
Before:

After:
image

Desktop View:
Mobile View:

### CODE REVIEW

File: test_file.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+aaaaaaaaaaaaaaaaaaa
+aaaaaaaaaaaaaaaaaaaaa
```
Comment (sayanta28) [inline test_file.txt:2]: - Why a is used?

**Review by sayanta28 — COMMENTED** [review]
Nice.

File: test_file.txt (hunk 1 above)
Comment (aniksikder6508) [reply test_file.txt:2]: good

Comment (mizanur0711) [reply test_file.txt:2]: see you not for mind !!

**Pull request merged**